src/nvim/indent.c

```diff
@@ -706,7 +706,7 @@ int get_lisp_indent(void)
 
             if (vi_lisp || ((*that != '"') && (*that != '\'')
                 && (*that != '#') && ((*that < '0') || (*that > '9')))) {
-              while (*that && (!vim_iswhite(*that) || quotecount || parencount)
+              while (*that && (!ascii_iswhite(*that) || quotecount || parencount)
```

Comment:
Line is more than 80 chars, which Travis doesn't like. Perhaps it could be split after the `&&`.